### ISSUE TITLE
Possible to mount in other locations?

### ISSUE THREAD

**Issue by vvvshel** [astrada/google-drive-ocamlfuse#836]
I have successfully mounted both My Drive ad Team Drive in home directory using the following command.
`google-drive-ocamlfuse [mountpoint] `
This command seems to mount google drive into home directory.

Is it possible to mount somewhere in other locations or directory?
 I might try to run a command like this : 

```
(sudo) mkdir /mnt/foldername
(sudo) google-drive-ocamlfuse  /mnt/foldername
```
Would it be working that way?

**Comment by GetPsyched** [comment]
Yeah, why not? The whole purpose of that parameter is for you to specify where do you want to mount Google Drive.

**Comment by vvvshel** [comment]
Can you suggest me the workable command to mount so that it can be assessible by a web app?

**Comment by GetPsyched** [comment]
You gave the command yourself the original message. It's `google-drive-ocamlfuse [mountpoint]`
